content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -625,6 +625,34 @@ Volume binding matrix for statically provisioned volumes:
 **Note:** Only statically provisioned volumes are supported for alpha release. Administrators should take care to consider these values when working with raw block devices.
 {{< /note >}}
 
+## Volume Snapshot and Restore Volume from Snapshot Support
+
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+
+Volume snapshot feature was added to support CSI Volume Plugins only. For details, see [volume snapshots](/docs/concepts/storage/volume-snapshots/).
+
+To enable support for restoring a volume from a volume snapshot data source, enable the
+`VolumeSnapshotDataSource` feature gate on the apiserver, controller-manager and the kubelet.
```

Comment:
I think kubelet does not need the feature gate?

Reply:
I think feature gate is needed by kubelet for some other features, but we can remove it from this one.